[2021-12-22 17:03:17,839] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: data_pipeline1.delete_past_images scheduled__2021-12-20T00:06:00+00:00 [queued]>
[2021-12-22 17:03:17,843] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: data_pipeline1.delete_past_images scheduled__2021-12-20T00:06:00+00:00 [queued]>
[2021-12-22 17:03:17,843] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-22 17:03:17,843] {taskinstance.py:1242} INFO - Starting attempt 1 of 4
[2021-12-22 17:03:17,843] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-22 17:03:17,852] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): delete_past_images> on 2021-12-20 00:06:00+00:00
[2021-12-22 17:03:17,853] {standard_task_runner.py:52} INFO - Started process 9155 to run task
[2021-12-22 17:03:17,855] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'data_pipeline1', 'delete_past_images', 'scheduled__2021-12-20T00:06:00+00:00', '--job-id', '17', '--raw', '--subdir', '/home/myamya/project/dockers/airflow/dags/hello_world.py', '--cfg-path', '/tmp/tmprrhg5hwp', '--error-file', '/tmp/tmpdxc_85sf']
[2021-12-22 17:03:17,855] {standard_task_runner.py:77} INFO - Job 17: Subtask delete_past_images
[2021-12-22 17:03:17,873] {logging_mixin.py:109} INFO - Running <TaskInstance: data_pipeline1.delete_past_images scheduled__2021-12-20T00:06:00+00:00 [running]> on host myamya
[2021-12-22 17:03:17,895] {taskinstance.py:1412} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=yangjae
AIRFLOW_CTX_DAG_ID=data_pipeline1
AIRFLOW_CTX_TASK_ID=delete_past_images
AIRFLOW_CTX_EXECUTION_DATE=2021-12-20T00:06:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-20T00:06:00+00:00
[2021-12-22 17:03:17,895] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2021-12-22 17:03:17,896] {subprocess.py:74} INFO - Running command: ['bash', '-c', '/bin/bash /home/myamya/project/image_files/delete.sh ']
[2021-12-22 17:03:17,900] {subprocess.py:85} INFO - Output:
[2021-12-22 17:03:17,903] {subprocess.py:93} INFO - Command exited with return code 0
[2021-12-22 17:03:17,916] {taskinstance.py:1270} INFO - Marking task as SUCCESS. dag_id=data_pipeline1, task_id=delete_past_images, execution_date=20211220T000600, start_date=20211222T080317, end_date=20211222T080317
[2021-12-22 17:03:17,946] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-22 17:03:17,956] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
